dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
This is scoped via config.json to packages.builds. The assumption is that these produces /consumes apis are all going to be interacted with via the run tool.  That is how it has been messaged in the conversations I've been a part of